ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -19,6 +19,7 @@
 import com.vungle.warren.VungleBanner;

```

Comment:
as we have copied some code as-is we should include a link to the FB code/file and add the license from that file to this Vungle src code file
https://github.com/googleads/googleads-mobile-android-mediation/blob/ebce3b3ccf1c7a0cd8ecb31819c0037b8885d584/ThirdPartyAdapters/facebook/facebook/src/main/java/com/google/ads/mediation/facebook/FacebookAdapter.java#L760

Reply:
fixed.